pages/cartPage/cartPage.html

```diff
@@ -0,0 +1,219 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Cart Page</title>
+    <link rel="stylesheet" href="">
+    <link rel="stylesheet" href="./cartPage.css">
+</head>
+
+<body>
+    <section class="page-wrap">
+        <section class="page-nav">
+            <nav class="dui-nav">
+                <a class="dui-nav__logo dui-util-txt-decoration-none" href="#">
+                    <h3 class="dui-util-fw-bld dui-util-clr-prim-p2">
+                        Garden Kart
+                    </h3>
+                </a>
+
+                <!-- Search -->
+                <div class="dui-nav__search dui-inp-txt dui-inp-txt--search">
+                    <i class="dui-inp-txt__icon dui-inp-txt__icon--search">
+                        <svg xmlns="http://www.w3.org/2000/svg"
+                            class="dui-inp-txt__icon-content dui-inp-txt__icon-content--search icon icon-tabler icon-tabler-search"
+                            width="56" height="56" viewBox="0 0 24 24" stroke-width="1.5" stroke="#059669" fill="none"
+                            stroke-linecap="round" stroke-linejoin="round">
+                            <path stroke="none" d="M0 0h24v24H0z" fill="none" />
+                            <circle cx="10" cy="10" r="7" />
+                            <line x1="21" y1="21" x2="15" y2="15" />
+                        </svg>
+                    </i>
+                    <input
```

Comment:
label tag

Reply:
I just merged it, will definitely make a follow-up PR to rectify it.

Reply:
Addressed the comment in the follow-up:
https://github.com/varun13169/gardenkart/pull/8